packages/components/drawer/src/index.ts

```diff
@@ -0,0 +1,14 @@
+export { type RootProps as DrawerProps } from './DrawerRoot'
+export { type ContentProps as DrawerContentProps } from './DrawerContent'
+export { type HeaderProps as DrawerHeaderProps } from './DrawerHeader'
+export { type BodyProps as DrawerBodyProps } from './DrawerBody'
+export { type FooterProps as DrawerFooterProps } from './DrawerFooter'
+export { type TriggerProps as DrawerTriggerProps } from './DrawerTrigger'
+export { type OverlayProps as DrawerOverlayProps } from './DrawerOverlay'
+export { type PortalProps as DrawerPortalProps } from './DrawerPortal'
+export { type TitleProps as DrawerTitleProps } from './DrawerTitle'
+export { type DescriptionProps as DrawerDescriptionProps } from './DrawerDescription'
+export { type CloseProps as DrawerCloseProps } from './DrawerClose'
+export { type CloseButtonProps as DrawerCloseButtonProps } from './DrawerCloseButton'
+
+export { Drawer } from './Drawer'
```

Comment:
I think that we should get aligned in how we name and export components.
- I usually name them as the file is named. For example: `DrawerTrigger` instead of `Trigger`
- I usually use this file to define the subcomponents. See example: https://github.com/adevinta/spark/blob/main/packages/components/input/src/index.ts

Reply:
Ok I will update